crates/util/src/node_with_source.rs

```diff
@@ -143,6 +143,10 @@ impl<'a> AstNode for NodeWithSource<'a> {
         CodeRange::new(self.node.start_byte(), self.node.end_byte(), self.source)
     }
 
+    fn full_source(&self) -> &str {
+        self.source
+    }
```

Comment:
This method was only necessary after fixing the last merge conflict. It is what it is :)